src/components/SinglePlayerGame.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react';
```

Comment:
So I saw in your game component you deleted this Single Player mode. Are you still going to incorporate this ?

Reply:
Yes, just moved it into a different component.